vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -92,15 +91,14 @@ if ($windowsSku -eq "2019") {
             "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.3.0",
             "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:v1.4.0",
             # Azure cloud node manager. Owner: nilo19 (Qi Ni)
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.1.14", # for k8s 1.22.x
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.24", # for k8s 1.23.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.30", # for k8s 1.23.x
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.11", # for k8s 1.24.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.17", # for k8s 1.24.x
```

Comment:
Is the old version 1.24.17 used in production? I think that we should keep it if it is used.

Reply:
No, it isn't.

Reply:
Thanks for the explanation. LGTM and will approve it after windows vhd build tests pass.